src/components/Column.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Column = ({
+  cards,
+  title
+}) => {
+
+  return (
+    <div className="column">
+      <h2>{title}</h2>
+      <ul>
+        {cards.map(card => (
+          <li key={card.id}>{card.title}</li>
+        ))}
+      </ul>
+    </div>
+  );
+};
+
+Column.propTypes = {
+  cards: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
```

Comment:
[proptypes] Let's describe the `shape` here when we decide on it.

Reply:
9fc0eb3